spec/models/call_off_contract_spec.rb

```diff
@@ -8,8 +8,16 @@
   describe "associations" do
     it { is_expected.to have_many(:participant_bands) }
 
-    it "is expected to have band_a with nil as the lowest min value" do
-      expect(call_off_contract.band_a.min).to eq(nil)
+    it "is expected to have band_a with the lowest minimum value" do
+      expect(call_off_contract.band_a.min.to_i).to eq(0)
+    end
+
+    it "is expected to have a total contract value" do
+      expect(call_off_contract.total_contract_value).to eq(2000 * 995) # recruitment_target * per_participant
+    end
+
+    it "is expected to have an uplift cap of 5% of the total contract value" do
+      expect(call_off_contract.uplift_cap).to eq(99_500.0)
```

Comment:
Same as above, 99500 doesn't make it easy to mentally verify that it's 5% of total contract value :P